.vscode/settings.json

```diff
@@ -0,0 +1,38 @@
+{
+  // Balancer frontend recommended settings
```

Comment:
JSON doesn't allow comments, does it? Should we include this? Maybe you could create a README in .vscode/ instead?

Reply:
Done